hazelcast/src/main/java/com/hazelcast/aggregation/impl/MaxAggregator.java

```diff
@@ -45,6 +45,9 @@ public void accumulate(I entry) {
     }
 
     private boolean isCurrentlyLessThan(R extractedValue) {
+        if (extractedValue == null) {
```

Comment:
isn't this `null` check needed by all other aggregators? i mean in `accumulate` methods of them. Because `AbstractAggregator#extract` can return `null` anytime ?